test/elixir/test/concurrent_writes_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule ConcurrentWritesTest do
+  use CouchTestCase
+
+  @moduletag :concurrent_writes
+  @moduletag kind: :single_node
+
+  @moduledoc """
+  Test CouchDB under concurrent write load
+  """
+
+  @tag :with_db
+  test "Primary data tests", context do
+    db_name = context[:db_name]
+    parent = self()
+    Enum.each(1..99,
```

Comment:
Minor nit: make 99 a variable at the top somewhere so we could make the test run with say 1000 or more docs if needed